### ISSUE TITLE
Move from phpunit 5.7 to 6.3 and php 5.5 to 7.1

### ISSUE THREAD

**Issue by jens1o** [alchemy-fr/BinaryDriver#26]
preparing for https://github.com/PHP-FFMpeg/PHP-FFMpeg/issues/412

--

- [ ] Set `declare(strict_types=1);` in *every* file and fix any type-converting struggles.
- [ ] Use scalar type hints, null type hints and return type values(even `void`) in any classes.
- [ ] Check whether we have any workarounds for older php versions
    - [ ] Remove them if they aren't necessary anymore.
- [ ] Convert any `array(...)` to `[...]`

**Comment by jens1o** [comment]
maybe we can update to PHPUnit 7, too.

**Comment by peter279k** [comment]
Is any updated commit for this issue?

**Comment by jens1o** [comment]
No, not yet. However, if you want to help out, you're welcome to file a Pull Request. Personally I should have time again after the final exam on Monday for ~8 weeks then.

**Comment by rickmills** [comment]
Any plans to merge in the pull request from August above? This is starting to cause massive headaches now that everyone else has updated to Symfony 4 packages.

**Comment by rickmills** [comment]
Any plans to merge in the pull request from August above? This is starting to cause massive headaches now that everyone else has updated to Symfony 4 packages.

@jygaulier @nmaillat @jens1o

**Comment by rickmills** [comment]
Any plans to merge in the pull request from August above? This is starting to cause massive headaches now that everyone else has updated to Symfony 4 packages.

@jygaulier @nmaillat @jens1o
